src/hiero_sdk_python/client/client.py

```
@@ -91,6 +92,7 @@ def get_node_account_ids(self) -> List[AccountId]:
         """
         Returns a list of node AccountIds that the client can use to send queries and transactions.
         """
+        d = 200;
```

Ruff (0.14.10)

Statement ends with an unnecessary semicolon

(E703)